src/index.js

```diff
@@ -124,11 +127,6 @@ class MySqlConnection {
     });
   }
 
-  releaseConnection() {
-    this.connection.release();
-    this.connection = null;
```

Comment:
Don't we want to set this.connection to null when it is released?

Reply:
I don't think there should be a `this.connection`.